packages/fvm/src/utils/convert.ts

```diff
@@ -57,3 +59,59 @@ export function toBigInt(value: string | number | bigint | Uint8Array): bigint {
       }
   }
 }
+
+/**
+ * Convert a base32 string to a hex string
+ * @param value - The base32 string to convert
+ * @param withPrefix - Whether to include the 0x prefix (default: true)
+ * @returns The hex string
+ */
+export function base32ToHex(value: string, withPrefix: boolean = true): string {
+  const bytes = base32.decode(value);
+  const hex = bytesToHex(bytes);
+  return withPrefix ? hex : hex.slice(2);
+}
+
+/**
+ * Convert a hex string to a base32 string
+ * @param value - The hex string to convert
+ * @param withPrefix - Whether to include the 0x prefix (default: true)
+ * @returns The base32 string
+ */
+export function hexToBase32(value: string, withPrefix: boolean = true): string {
+  const bytes = hexToBytes(withPrefix ? (value as Hex) : `0x${value}`);
+  return base32.encode(bytes);
+}
+
+/**
+ * Compare two Uint8Arrays
+ * @param a - The first Uint8Array
+ * @param b - The second Uint8Array
+ * @returns The result of the comparison
+ */
+export function compareUint8Arrays(a: Uint8Array, b: Uint8Array): number {
```

Comment:
note: this was copied from the `uint8arrays` package and tweaked for ts compiler errors